set.mm

```diff
@@ -425498,66 +425636,60 @@ that the number n of vertices in G is exactly k(k-1)+1.".  Variant of
     $( Lemma 2 for ~ numclwwlkovf2ex :  Transformation of a walk and two edges
        into a walk extended by two vertices/edges.  (Contributed by AV,
        22-Sep-2018.)  (Revised by AV, 27-Jan-2022.) $)
-    numclwwlkovf2exlem2 $p |- ( ( ( ( G e. USGraph /\ X e. V
-                                      /\ N e. ( ZZ>= ` 3 ) )
+    numclwwlkovf2exlem2 $p |- ( ( ( ( X e. V /\ Y e. V /\ N e. ( ZZ>= ` 3 ) )
                    /\ ( ( W e. Word V /\ A. i e. ( 0 ..^ ( ( # ` W ) - 1 ) )
                                          { ( W ` i ) , ( W ` ( i + 1 ) ) } e. E
                    /\ { ( lastS ` W ) , ( W ` 0 ) } e. E )
                    /\ ( # ` W ) = ( N - 2 ) /\ ( W ` 0 ) = X ) )
-                   /\ Y e. ( G NeighbVtx X ) )
+                   /\ { X , Y } e. E )
```

Comment:
By the way: I presume the equivalence of these two lines (i.e., under the required graphness and sethood antecedents,  Y e. ( G NeighbVtx X ) <-> { X , Y } e. E ) is already in Main ? As well as  X e. ( G NeighbVtx Y ) <-> Y e. ( G NeighbVtx X ) ?

Reply:
Y e. ( G NeighbVtx X ) <-> { X , Y } e. E ) : ~nbupgrel
X e. ( G NeighbVtx Y ) <-> Y e. ( G NeighbVtx X ) : ~nbgrsym